src/main/java/com/gooddata/notification/Subscription.java

```diff
@@ -85,7 +110,13 @@ public TriggerCondition getCondition() {
 
     @JsonProperty("message")
     public MessageTemplate getTemplate() {
-        return template;
+        return messageTemplate;
+    }
+
+    @JsonProperty("subject")
+    @JsonInclude(NON_NULL)
```

Comment:
Please move the `@JsonInclude(NON_NULL)` to the class level. It's more common here in SDK as well as we have `notNull` checks for other properties in constructor so they cannot be null during serialization.